src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +27,13 @@ public interface MachineService {
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<? super Machine> machines, Machine value);
```

Comment:
The `fill` method should use the type parameter `T` instead of the raw type `Machine` for the `machines` list and `value`. For example: `void fill(List<? super T> machines, T value);`